src/generator/AutoRest.AzureResourceSchema/CodeGeneratorArs.cs

```diff
@@ -29,12 +29,35 @@ public CodeGeneratorArs()
             {
                 StringWriter stringWriter = new StringWriter();
                 ResourceSchemaWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
-                await Write(stringWriter.ToString(), resourceProvider + ".json", true);
+                await Write(stringWriter.ToString(), resourceProvider + Path.DirectorySeparatorChar + resourceProvider + ".json", true);
 
                 stringWriter = new StringWriter();
-                ResourceMarkdownWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
-                await Write(stringWriter.ToString(), resourceProvider + ".md", false);
+                var md = ResourceMarkdownGenerator.Generate(resourceSchemas[resourceProvider]);
+
+                foreach (var m in md)
+                {
+                    var content = m.Content.Replace("\"boolean\"", "boolean");
```

Comment:
Ah, here is this difference. Why is this replace happening? I thought that Boolean values themselves i.e. `true` and `false` didn't have to be in quotes, but I assumed the value `boolean` would